subsys/emul/espi/emul_espi_host.c

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Emulator for the generic eSPI Host. This supports basic
+ * host operations.
+ */
+
+#define DT_DRV_COMPAT zephyr_espi_emul_espi_host
+
+#define LOG_LEVEL CONFIG_ESPI_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(espi_host);
+
+#include <device.h>
+#include <emul.h>
+#include <drivers/espi.h>
+#include <drivers/espi_emul.h>
+
+struct vw_data {
+	enum espi_vwire_signal sig;
+	uint8_t level;
```

Comment:
I still don't know what a level it from this change

Reply:
Commented.